build.gradle

```diff
@@ -19,6 +19,7 @@ buildscript {
     classpath "io.codearte.gradle.nexus:gradle-nexus-staging-plugin:0.20.0"
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
     classpath 'com.android.tools.build:gradle:3.5.0'
+    classpath 'com.google.gms:google-services:4.3.3'
```

Comment:
Does this introduce the requirement to have Google Play Services installed on devices? I'm not sure how it works.

Reply:
Yes this does, doe that introduce other complications?